scripts/benchmark/README.md

```diff
@@ -0,0 +1,33 @@
+Script to run an executable (for instance JBMC) on a list of Java methods.
+
+# Install
+
+    npm install
```

Comment:
Actually that doesn't work for me, it complains about a missing `package.json` file.

Reply:
Sorry I forgot to add the file, should now be ok.